packages/scenario/test/bind-types.test.ts

```diff
@@ -0,0 +1,647 @@
+		 */
+		describe('jr:count computed from a calculate expressions', () => {
+			it('casts a fractional value computed from a node with type="string"', async () => {
+				const scenario = await Scenario.init(
+					'Cast fractional value to int',
+					// prettier-ignore
+					html(
+						head(
+							title('Cast fractional value to int'),
+							model(
+								mainInstance(
+									t('data id="cast-fractional-value-to-int',
+										t('count-calc-input-str'),
+										t('count-calc'),
+										t('repeat-count jr:template=""'))),
+								bind('/data/count-calc-input-str').type('string'),
+								bind('/data/count-calc')
+									.type('int')
+									.calculate('/data/count-calc-input-str'))),
+						body(
+							input('/data/count-calc-input-str'),
+							repeat('/data/repeat-count', '/data/count-calc')))
+				);
+
+				scenario.answer('/data/count-calc-input-str', '3.5');
+
+				expect(scenario.answerOf('/data/count-calc')).toEqualAnswer(intAnswer(3));
+				expect(scenario.countRepeatInstancesOf('/data/repeat-count')).toBe(3);
+			});
+
+			it('casts a fractional value computed from a node with type="decimal"', async () => {
+				const scenario = await Scenario.init(
+					'Cast fractional value to int',
+					// prettier-ignore
+					html(
+						head(
+							title('Cast fractional value to int'),
+							model(
+								mainInstance(
+									t('data id="cast-fractional-value-to-int',
+										t('count-calc-input-dec'),
+										t('count-calc'),
+										t('repeat-count jr:template=""'))),
+								bind('/data/count-calc-input-dec').type('decimal'),
+								bind('/data/count-calc')
+									.type('int')
+									.calculate('/data/count-calc-input-dec'))),
+						body(
+							input('/data/count-calc-input-dec'),
+							repeat('/data/repeat-count', '/data/count-calc')))
+				);
+
+				scenario.answer('/data/count-calc-input-dec', 4.5);
+
+				expect(scenario.answerOf('/data/count-calc')).toEqualAnswer(intAnswer(4));
+				expect(scenario.countRepeatInstancesOf('/data/repeat-count')).toBe(4);
+			});
+
+			it('casts a fractional value computed from a number literal', async () => {
+				const scenario = await Scenario.init(
+					'Cast fractional value to int',
+					// prettier-ignore
+					html(
+						head(
+							title('Cast fractional value to int'),
+							model(
+								mainInstance(
+									t('data id="cast-fractional-value-to-int',
+										t('count-calc'),
+										t('repeat-count jr:template=""'))),
+								bind('/data/count-calc')
+									.type('int')
+									.calculate('5.5'))),
+						body(
+							repeat('/data/repeat-count', '/data/count-calc')))
+				);
+
+				expect(scenario.answerOf('/data/count-calc')).toEqualAnswer(intAnswer(5));
+				expect(scenario.countRepeatInstancesOf('/data/repeat-count')).toBe(5);
+			});
+		});
+		it('casts a calculated value to an int', async () => {
+			// const scenario = await Scenario.init(
```

Comment:
Why is this one commented out?

Reply:
I think it should've been deleted! 🙃 It was one test which became the basis for the three above.